migrations/Version202608041200001488_taoAdvancedSearch.php

```
@@ -0,0 +1,67 @@
+        foreach (self::LEGACY_SERVICE_IDS as $serviceId) {
+            if ($serviceManager->has($serviceId)) {
+                $serviceManager->unregister($serviceId);
+            }
+        }
+
+        $this->addReport(
+            Report::createSuccess('Legacy Item Comment ServiceManager configs unregistered')
+        );
+    }
+
+    public function down(Schema $schema): void
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$schema'. (undefined)

(UnusedFormalParameter)